bundles/org.dataflowanalysis.analysis/src/org/dataflowanalysis/analysis/core/AbstractVertex.java

```diff
@@ -126,40 +126,86 @@ public boolean isSource() {
     /**
      * Returns a list of node characteristic with the given characteristic type
      * <p>
-     * See {@link AbstractVertex#getDataFlowVariablesWithName(String)} for a similar method for data flow
+     * See {@link AbstractVertex#getDataFlowVariables(String)} for a similar method for data flow
      * variables
      * @param characteristicType Name of the characteristic type
      * @return Returns a list of all node characteristics matching the characteristic type
      */
-    public List<CharacteristicValue> getNodeCharacteristicsWithName(String characteristicType) {
+    public List<CharacteristicValue> getNodeCharacteristics(String characteristicType) {
```

Comment:
Should it not be called getNodeCharaceristicsWithType? From the name alone I would expect an aggregated list of CharacteristicValues from all types and not based on the type.

Reply:
Fixed in 16e2185304148ef0a99a015f4b6a125baa02e335